core/src/main/java/org/bitcoinj/core/BitcoinSerializer.java

```diff
@@ -231,7 +231,7 @@ private Message makeMessage(String command, int length, byte[] payloadBytes, byt
         } else if (command.equals("getutxos")) {
             return new GetUTXOsMessage(params, payloadBytes);
         } else if (command.equals("sendheaders")) {
-            return new SendHeadersMessage(params, payloadBytes);
+            return new SendHeadersMessage();
```

Comment:
This change seems unrelated.

Reply:
Another one that fell through the cracks.